ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/components/ExceptionRequestModal.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Modal, pluralize } from '@patternfly/react-core';
+
+import { CveExceptionRequestType } from '../types';
+
+export type ExceptionRequestModalProps = {
+    type: CveExceptionRequestType;
+    cves: string[];
+    onClose: () => void;
+};
+
+function ExceptionRequestModal({ type, cves, onClose }: ExceptionRequestModalProps) {
+    const cveCountText = pluralize(cves.length, 'workload CVE');
+    const title =
+        type === 'DEFERRAL'
+            ? `Request deferral for ${cveCountText}`
+            : `Mark ${cveCountText} as false positive`;
```

Comment:
* Plus: This pair has as parallel grammar as practical.
* Minus: Inconsistent with items above which would be **Request deferral for CVE** or **CVEs** plural.